canonical/cos-tool
# Fix error with assume-no-moving-gc

## Issue


This PR fixes https://github.com/canonical/cos-tool-snap/issues/1

## How to test it:

1. Build `cos-tool`: ` go build --ldflags '-extldflags "-static"'`
2. Run it to check [the error](https://github.com/canonical/cos-tool-snap/issues/1) is gone:

```shell
$ ./cos-tool 

NAME:
   cos-tool - Validates Prometheus and Loki expressions, adds Juju Topology to label matchers

USAGE:
   cos-tool [global options] command [command options] [arguments...]

COMMANDS:
   transform, t                          
   validate-rules, v, lint, l, validate  
   validate-config                       
   help, h                               Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --format promql|logql, -f promql|logql  Inject expressions into promql|logql (default: "promql")
   --help, -h  
```
